resources/views/poem.blade.php

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="<?php echo e(str_replace('_', '-', app()->getLocale())); ?>">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Kirby Marquez Poetry">
+    <title>Kirby Marquez | Poems</title>
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link rel="stylesheet" href="/css/app.css">
+  </head>
+  <body>
+    <div class="poem-container">
+    <div class="poem-content-container">
+        <a href="/" class="back-button"></a>
+        <div class="poem-title">{{ $poem->title ?? '' }}</div>
+        <div class="poem-body">{{ $poem->body ?? '' }}</div>
```

Comment:
I don't think you need `?? ''` for these. As far as I can tell from the [migration](https://github.com/markwuu/kirby-site/blob/master/database/migrations/2020_09_06_011830_create_poems_table.php), `title` and `body` are both required. Even if they were nullable, a `null` value is rendered as an empty string, so the fallback wouldn't be necessary.